homeassistant/components/romy/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "romy",
+  "name": "ROMY Vacuum Cleaner",
+  "codeowners": ["@xeniter"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/romy",
+  "iot_class": "local_polling",
+  "requirements": ["romy==0.0.5"],
```

Comment:
I saw that https://github.com/xeniter/romy/tags does not list this version. Can you tag all releases on github and provide a link to the changes or changelog in the PR descripton?
This is an example of a typical PR that contains a dependency bump and what is expected: https://github.com/home-assistant/core/pull/105462

Reply:
forgot to push the tag yesterday, pushed it now + added some description in the PR